addons/essentials/src/index.ts

```diff
@@ -18,7 +18,14 @@ const requireMain = (configDir: string) => {
     // eslint-disable-next-line global-require,import/no-dynamic-require
     main = require(mainFile);
```

Comment:
@yannbf I wonder whether we want to use `serverRequire` here instead?

Reply:
Oh interesting, are all of these extensions possible for `.storybook/main` files? https://github.com/storybookjs/storybook/blob/fe0920c06aa6769f912ea0a371f66f4fa42c609a/lib/core-common/src/utils/interpret-files.ts#L4

Reply:
Yup!

Reply:
Cool, that simplifies things a bit.  I've also used `serverRequire` in a few of the other addons as well, since I was getting warning messages from them as well.